.github/workflows/c-releases.yml

```
@@ -50,4 +55,5 @@ jobs:
         with:
           name: 'Kuliya latest'
           artifacts: 'c-release.tar.gz'
+          tag: ${{ inputs.tag }}
```

Maybe enforce the `c-` prefix here, and update the description at the top?

```suggestion
          tag: "c-${{ inputs.tag }}"
```

---

I agree